spec/services/atmosphere/vm_updater_spec.rb

```diff
@@ -23,19 +23,23 @@
   subject { Atmosphere::VmUpdater.new(cs, server, updater_class) }
 
   describe 'VM states' do
-    context "when task state equals to image_snapshot" do
-      let(:server) do
-        server_double(state: 'active',
-                      task_state: 'image_snapshot',
-                      flavor: {'id' => "1"})
-      end
+    it 'is saving when task state equals to image_snapshot' do
+      saving_when_task_state('image_snapshot')
+    end
 
-      subject { Atmosphere::VmUpdater.new(cs, server, updater_class) }
+    it 'is saving when task state equals to image_pending_upload' do
+      saving_when_task_state('image_pending_upload')
+    end
 
-      it 'sets "saving" state' do
-        vm = subject.update
-        expect(vm.state).to eq 'saving'
-      end
+    def saving_when_task_state(state)
+      server = server_double(state: 'active',
+                             task_state: state,
+                             flavor: {'id' => "1"})
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.